game/script-topics.rpy

```diff
@@ -4220,113 +4220,293 @@ init 5 python:
+            n 1usqsm "Anyway,{w=0.2} who knows?"
+            n 1fsqsm "Maybe you'll get to see some of my handiwork some day,{w=0.2} [player]."
+            n 1fsqbg "I bet you'd like that,{w=0.2} huh?{w=0.5}{nw}"
+            extend 1fchsml " Ehehe."
 
-        if Natsuki.isAffectionate(higher=True):
-            n 1fsqbg "Besides,{w=0.1} [player].{w=0.2} You seem to have pretty good taste."
-            n 1fsqsml "I think I can trust your judgement..."
+            # Continue to unlock dialogue
 
-        return
+    elif Natsuki.isAffectionate(higher=True):
+        if already_unlocked_cosplay_outfits:
+            n 1tsqpu "Huh?{w=0.5}{nw}"
+            extend 1tsqsf " Cosplay {i}again{/i}, [player]?"
 
-    elif Natsuki.isNormal(higher=True):
-        n 1unmaj "Cosplay,{w=0.1} huh?"
-        n 1ulraj "Well...{w=0.3} I mean,{w=0.1} I've considered it,{w=0.1} if that's what you're asking."
-        n 1nnmbo "I never really thought about it that much until I got more into manga and things like that."
-        n 1flrbg "It kinda feels like once you start getting into that stuff,{w=0.1} you discover tons more at once!"
-        n 1nnmaj "But anyway,{w=0.1} I've never actually gone out and cosplayed myself."
-        n 1flleml "T-{w=0.1}that isn't to say there's anything stopping me,{w=0.1} of course!"
+            if Natsuki.isWearingClothes("jn_clothes_trainer_cosplay") or Natsuki.isWearingClothes("jn_clothes_sango_cosplay"):
+                n 1fsqsflsbl "...Was {i}wearing{/i} it seriously not enough already?"
+                n 1fsqsslsbl "Or did I somehow awaken some kind of hidden nerdiness in you?"
+                n 1fslsslsbr "Ehehe..."
+                n 1fcsemlsbr "A-{w=0.3}anyway!{w=0.5}{nw}"
 
-        if already_mentioned_sewing:
-            n 1fllss "I told you already that I'm pretty good with a needle and thread,{w=0.1} so that's a-{w=0.1}okay!"
+            else:
+                n 1tlraj "I gotta say,{w=0.2} I'm actually kinda impressed."
+                n 1fslsslsbr "Sucking up to my interest in manga,{w=0.75}{nw}"
+                extend 1fsqsslsbr " bugging me about cosplay..."
+                n 1tsqsm "You're absorbing my sense of taste pretty fast,{w=0.2} huh?{w=0.75}{nw}"
+                extend 1uchgn " Like a little dorky sponge!"
+                n 1fchsm "Ehehe."
+                n 1fchss "Anyway!{w=0.75}{nw}"
+
+            extend 1nllss " Putting all that aside..."
+            n 1unmaj "I actually wouldn't be {i}against{/i} doing a little more cosplay at all."
+            n 1ulrpu "I mean,{w=0.75}{nw}"
+            extend 1flrss " I've already got most of what I need."
+            n 1fsqss "{i}Amazing{/i} ideas?{w=0.75}{nw}"
+            extend 1fcsbg " Check!{w=0.75}{nw}"
+            extend 1tsqsm " Tons of experience?{w=0.75}{nw}"
+            extend 1fchbg " Check!"
+            n 1usqgn "{i}Unmatched{/i} handiwork with a needle and threat?{w=1}{nw}"
```

Comment:
```suggestion
            n 1usqgn "{i}Unmatched{/i} handiwork with a needle and thread?{w=1}{nw}"
```

Reply:
Might want to change that

Reply:
I swear I had this happen before

Thanks!